db/migrate/20221214214107_add_fields_to_travel.rb

```diff
@@ -0,0 +1,13 @@
+class AddFieldsToTravel < ActiveRecord::Migration[6.1]
+  def change
+    rename_column :travels, :transport_needed, :arrival_transport
+    rename_column :travels, :carrier, :arrival_carrier
+    rename_column :travels, :route_num, :arrival_route_num
+    add_column :travels, :arrival_date, :date
+    add_column :travels, :arrival_time, :time
+    add_column :travels, :depart_transport, :string
+    add_column :travels, :depart_route_num, :time
```

Comment:
fixed in later migration

Reply:
had to run annotate, fixed here [aa61a19](https://github.com/lsa-mis/mmss-mysql/pull/107/commits/aa61a19dd8c7be6b40aefd5b2939a7a2c058621e)